### DISCUSSION TITLE
Supplier definition in GTC

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#13351] [Q&A] [answered]
Weblate's [General Terms and Conditions](https://weblate.org/en/terms/) has "Supplier" mentioned 29 times across Supplier's and User's rights and obligations (sections 8 and 9 fo the GTC), however it has no term definitions in the item 1.2.

Should Supplier definition be included?

Or maybe replaced with "Provider" ?

**Comment by nijel** [comment, marked as answer]
Good catch (again), "Provider" should have been used instead of "Supplier" (as that is the term that is defined in the preamble).

So many people were reviewing this and nobody noticed it so far :-(.

Fixed in https://github.com/WeblateOrg/wllegal/commit/bf5e83a240e5eabcfb608e86657f7df6b47c1828.

**Reply by rffontenelle** [reply]
Hmm it seems "Provider‘s Rights and Obligations" is using opening single quotes instead of closing single quotes. Sorry for not noticing before

**Reply by nijel** [reply]
Thanks, fixed that one as well.
